src/main/java/uk/gov/hmcts/darts/retention/controller/RetentionController.java

```diff
@@ -33,20 +33,22 @@ public class RetentionController implements RetentionApi {
 
     private final RetentionPostService retentionPostService;
 
+    @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
     @Override
     public ResponseEntity<List<GetCaseRetentionsResponse>> retentionsGet(Integer caseId) {
         return new ResponseEntity<>(retentionService.getCaseRetentions(caseId), HttpStatus.OK);
     }
 
     @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
     @Authorisation(contextId = ContextIdEnum.CASE_ID, bodyAuthorisation = true,
-            securityRoles = {JUDGE, REQUESTER, APPROVER, TRANSCRIBER, TRANSLATION_QA, ADMIN},
-            globalAccessSecurityRoles = {JUDGE, ADMIN})
+        securityRoles = {JUDGE, REQUESTER, APPROVER, TRANSCRIBER},
```

Comment:
is SUPER_ADMIN not relevant here?

Reply:
My understanding of the controller authorisation is that the `securityRoles` list the roles that it is necessary to check against a `contextId` e.g. case_id param

According to our Authorisation business logic (DARTS All user permissions.xlsx), "Super Admin" Can access all courthouses. So we wouldn't expect the security_group to have any association with courthouse.
That's what the meaning behind `global_access	true` on the security_group is about.
So the global role is present in the other list that Karen added to the annotation called `globalAccessSecurityRoles`.

`SELECT grp_id, cth_id FROM security_group_courthouse_ae WHERE grp_id = 1;`

security_group
--------------
grp_id	1
rol_id	11
group_name	SUPER_ADMIN
global_access	true